app/assets/stylesheets/pages/about.scss

```diff
@@ -1,21 +1,36 @@
-.about-banner {
-  @extend .banner;
-  @include animation(FadeIn 2s ease-in);
-  @include background(linear-gradient(rgba(255, 255, 255, 0.3),
-                      rgba(255, 255, 255, 0.3)),
-                      image-url("about/about.jpg"));
+#about-blackener {
+  margin-top: -300px;
+  height: 300px;
+}
+
+#about-image {
+  position: fixed;
```

Comment:
Properties should be ordered background, background-position, background-repeat, background-size, height, position, top, width, z-index